src/dscanner/analysis/auto_ref_assignment.d

```diff
@@ -9,6 +9,8 @@ import dparse.lexer;
 import dparse.ast;
 import dscanner.analysis.base;
 
+
+
```

Comment:
Maybe get rid of these unrelated changes?

Reply:
Done.